src/popup/components/InputField.vue

```diff
@@ -158,11 +158,20 @@ export default {
     },
   },
   methods: {
+    checkModifierKey(event) {
```

Comment:
In my opinion there is no need to create separate method for this logic as it's used only once.